### PR TITLE
Event reg confirmation message fix

### PR DESCRIPTION
**Pull request by martypowell**
Fixed a missing null check that caused an error when registering.

### CODE REVIEW

File: js/apps/events-page/controllers/eventRegistrationCtrl.js [hunk 1]
```diff
@@ -30,20 +30,20 @@
 				vm.postResult = postResult.data;
 
 				const data = vm.postResult.Data;
-				const hasConfirmationMessage = data && 
-					Object.prototype.hasOwnProperty.call(vm.postResult.Data, 'ConfirmationMessage') &&
+				const hasConfirmationMessage = data &&
+                    Object.prototype.hasOwnProperty.call(data, 'ConfirmationMessage') &&
+                    data.ConfirmationMessage &&
```
Comment (ascire) [inline js/apps/events-page/controllers/eventRegistrationCtrl.js]: Indentation here is wacky.

Comment (martypowell) [reply js/apps/events-page/controllers/eventRegistrationCtrl.js]: yea no kidding, looking itno this.

File: js/apps/events-page/controllers/eventRegistrationCtrl.js [hunk 2]
```diff
@@ -30,10 +30,8 @@
 				vm.postResult = postResult.data;
 
 				const data = vm.postResult.Data;
-				const hasConfirmationMessage = data &&
-                    Object.prototype.hasOwnProperty.call(data, 'ConfirmationMessage') &&
-                    data.ConfirmationMessage &&
-					data.ConfirmationMessage.length;
+				const hasConfirmationMessage = data && Object.prototype.hasOwnProperty.call(data, 'ConfirmationMessage') &&
+                    data.ConfirmationMessage && data.ConfirmationMessage.length;
```
Comment (ascire) [inline js/apps/events-page/controllers/eventRegistrationCtrl.js]: Not gonna lie... This somehow looks worse.  Is there a mix of tabs and spaces or something?

Comment (martypowell) [reply js/apps/events-page/controllers/eventRegistrationCtrl.js]: @ascire yes something is up with my IDE

**Comment by martypowell** [comment]
@ascire ready for another look, i just one lined it for now since I couldn't get my editor to line it up properly

**Pull request merged**